### ISSUE TITLE
Allow VariableBuffer to take in BufferParameters?

### ISSUE THREAD

**Issue by WindingWinter** [NetTopologySuite/NetTopologySuite#696]
Similar to [Buffer](https://nettopologysuite.github.io/NetTopologySuite/api/NetTopologySuite.Operation.Buffer.BufferOp.html#NetTopologySuite_Operation_Buffer_BufferOp_Buffer_NetTopologySuite_Geometries_Geometry_System_Double_NetTopologySuite_Operation_Buffer_BufferParameters_), I think VariableBuffer should also be able to  take in BufferParameters as an argument. 

This is because I want to use one side buffering for VariableBuffer also. The method [Buffer(Geometry, Double[])](https://nettopologysuite.github.io/NetTopologySuite/api/NetTopologySuite.Operation.Buffer.VariableBuffer.html#NetTopologySuite_Operation_Buffer_VariableBuffer_Buffer_NetTopologySuite_Geometries_Geometry_System_Double___) would be an ideal extension to take in the BufferParameters

**Comment by bjornharrtell** [comment]
Seems stale to me, let me know if still of interest.

**Comment by WindingWinter** [comment]
Yes, I'm still interested in this

**Comment by bjornharrtell** [comment]
@WindingWinter ok - may be something that should/could be brought upstream. But might look into if it's possible to do without deviation.

**Comment by WindingWinter** [comment]
Upstream means? At JTS?

**Comment by bjornharrtell** [comment]
Yes NTS upstream is JTS. If this capability is/was in upstream it would be a good candidate to port as is. If it's not, we need to evaluate if we want to extend NTS to include it even if it's not in upstream.

**Comment by bjornharrtell** [comment]
This does not exist in JTS. I suggest to bring up the feature request or contribute it there.
